docs/learn/learn-offenses.md

```diff
@@ -0,0 +1,226 @@
+| Polkadot | 774  | Small Equivocation | [The validator](https://matrix.to/#/!NZrbtteFeqYKCUGQtr:matrix.parity.io/$165562246360408hKCfC:matrix.org?via=matrix.parity.io&via=corepaper.org&via=matrix.org) migrated servers and cloned the keystore folder. The on-chain event can be viewed [here](https://polkadot.subscan.io/extrinsic/11190109-0?event=11190109-5). | The validator did not submit a request for the slash to be canceled.                                                               |
+| Kusama   | 3329 | Small Equivocation | The validator operated a test machine with cloned keys; the test machine was online at the same time as the primary, which resulted in a slash. Details can be found [here](https://kusama.polkassembly.io/post/1343).                                                                                                        | The validator requested a slash cancellation, but the council declined.                                                            |
+| Kusama   | 3995 | Small Equivocation | The validator noticed several errors, after which the client crashed, and a slash was applied. The validator recorded all events and opened GitHub issues to allow for technical opinions to be shared. Details can be found [here](https://kusama.polkassembly.io/post/1733).                                                | The validator requested to cancel the slash. The council approved the request as they believed the error was not operator-related. |
+
+#### Slashing Across Eras
+
+There are three main difficulties to account for with slashing in NPoS:
+
+- A nominator can nominate multiple validators and be slashed via any of them.
+- Until slashed, the stake is reused from era to era. Nominating with N coins for E eras in a row
+  does not mean you have N\*E coins to be slashed - you've only ever had N.
+- Slashable offenses can be found after the fact and out of order.
+
+To balance this, we only slash for the maximum slash a participant can receive in some time period
+rather than the sum. This ensures protection from overslashing. Likewise, the period over which
+maximum slashes are computed is finite, and the validator is chilled with nominations withdrawn
+after a slashing event, as stated in the previous section. This prevents rage-quit attacks in which,
+once caught misbehaving, a participant deliberately misbehaves more because their slashing amount is
+already maxed out.
+
+### Disabling
+
+**Disabling** stops validators from performing specific actions. Disabling is further divided into:
+
+- On-chain disabling lasts for a whole era and stops validators from block authoring, backing
+  through runtime filtering, initiating a dispute, and backing. It also makes other nodes ignore
+  backing statements.
```

Comment:
```suggestion
- On-chain disabling lasts for a whole era and stops validators from block authoring, backing and initiating a dispute.
```

deduplication + simplified. Backing statement filtering is there but it is an optimization that is not worth mentioning.